lua/example_of_dynamic_config_request.lua

```diff
@@ -5,4 +5,43 @@
 -- Functions to process the GT policies.
 -- ......
 
-return "A default message from the dynamic configuration block!"
+local dylib = require("dylib")
+local ffi = require("ffi")
+
+local dyc = gatekeeper.c.get_dy_conf()
+
+-- Setup Grantor Prefixes.
+local ret = dylib.c.add_fib_entry_lock("128.197.40.100/24",
+	dylib.c.GK_FWD_GATEWAY_BACK_NET, "10.0.1.254", dyc.gk)
+if ret < 0 then
+	return "gk: failed to add an FIB entry\n"
+end
+
+-- Setup Prefixes that need to forward to the back network.
+ret = dylib.c.add_fib_entry_lock("187.73.40.0/30",
+	dylib.c.GK_FWD_GRANTOR, "128.197.40.100", dyc.gk)
+if ret < 0 then
+	return "gk: failed to add an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("187.73.40.0/30", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("128.197.40.100/32", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("128.197.40.100/24", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+ret = dylib.c.del_fib_entry_lock("10.0.1.254/32", dyc.gk)
+if ret < 0 then
+	return "gk: failed to delete an FIB entry\n"
+end
+
+return "gk: successfully processed all the FIB entries\n"
```

Comment:
The example configuration (1) tries to setup the Grantor prefixes ``128.197.40.100/24``, and the gateway ``10.0.1.254`` will be responsible for forwarding the packets to these grantors; (2) setup the destination IP prefix ``187.73.40.0/30``, all the packets with these destinations should be forwarded to grantor ``128.197.40.100``. (3) delete the FIB entry for the destination IP prefix ``187.73.40.0/30``; delete the FIB entry for the grantor ``128.197.40.100``; delete the FIB entry for the grantor prefix ``128.197.40.100/24``; delete the FIB entry for the gateway ``10.0.1.254``; all the deletions should be successful.